packages/test-app/tests/integration/components/course/header-test.js

```diff
@@ -3,10 +3,10 @@ import { setupRenderingTest } from 'test-app/tests/helpers';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import { a11yAudit } from 'ember-a11y-testing/test-support';
-import { component } from 'ilios-common/page-objects/components/course-header';
+import { component } from 'ilios-common/page-objects/components/course/header';
 import { setupMirage } from 'ember-cli-mirage/test-support';
 
-module('Integration | Component | course-header', function (hooks) {
+module('Integration | Component | course/header', function (hooks) {
```

Comment:
Now you're just showing off! (would never have occurred to me to fix this too 😆)